slave-base/contrib/bin/run-jnlp-client

```diff
@@ -73,6 +73,11 @@ if [[ $# -gt 1 ]]; then
   CONTAINER_MEMORY_IN_BYTES=`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`
   DEFAULT_MEMORY_CEILING=$((2**40-1))
   if [ "${CONTAINER_MEMORY_IN_BYTES}" -lt "${DEFAULT_MEMORY_CEILING}" ]; then
+    # set this JVM's -Xmx and -Xms if not set already (not propagated to any
+    # child JVMs).  -Xmx can be calculated as a percentage, capped to a maximum,
+    # or specified straight.  -Xms can be calculated as a percentage or
+    # specified straight.  For the JNLP slave by default we specify -Xmx of 50%,
+    # uncapped; -Xms unspecified (JVM default is 1/64 of -Xmx).
```

Comment:
ah I'm reviewing commit by commit  ... this comment is probably sufficient wrt my ask for a comment elsewhere, unless @jim-minter feels so inclined